app/__init__.py

```diff
@@ -10,9 +12,12 @@
 
 
 def create_app(environment):
-    application = Flask('api')
+    application = Flask('app')
```

Comment:
document-download?

Reply:
That's what I wanted to do at first, but almost all Notify apps set it to 'app'. I think it's only used for the logger name, so 'app' makes sense.